### PR TITLE
doc: add ttlSecondsAfterFinished example

### PR DESCRIPTION
**Pull request by googs1025**
Fixes https://github.com/kubernetes-sigs/jobset/issues/519

Change summary:

1. add ttl example yaml file
2. Modify version release-0.4 to release-0.5

### CODE REVIEW

File: site/content/en/docs/tasks/_index.md [hunk 1]
```diff
@@ -19,22 +19,24 @@ no_list: true
...
 
-- [Startup Policy](https://github.com/kubernetes-sigs/jobset/blob/release-0.4/examples/startup-policy/startup-driver-ready.yaml) demonstrates how we can define a startup order for ReplicatedJobs in order to ensure a "leader"
+- [Startup Policy](https://github.com/kubernetes-sigs/jobset/blob/release-0.5/examples/startup-policy/startup-driver-ready.yaml) demonstrates how we can define a startup order for ReplicatedJobs in order to ensure a "leader"
 pod is running before the "workers" are created. This is important for enabling the leader-worker paradigm in distributed ML training, where the workers will attempt to register with the leader as soon as they spawn.
 
+- [Automatic Cleanup for Finished Jobs](https://github.com/kubernetes-sigs/jobset/blob/release-0.5/examples/simple/ttl-after-finished.yaml) demonstrates an example of `ttlSecondsAfterFinished`. ttlSecondsAfterFinished is a feature that allows you to specify the maximum lifetime of a job. When a job reaches the specified lifetime, it will be deleted.
```
Comment (danielvegamyhre) [inline site/content/en/docs/tasks/_index.md:35]:
```suggestion
- [TTL after finished](https://github.com/kubernetes-sigs/jobset/blob/release-0.5/examples/simple/ttl-after-finished.yaml) demonstrates how you can configure a JobSet to be cleaned up automatically after a defined period of time has passed after the JobSet finishes.
```

Comment (googs1025) [reply site/content/en/docs/tasks/_index.md]: done

**Pull request merged**